giraffe/src/utils/fromFlux.ts

```diff
@@ -158,14 +178,16 @@ export const fromFlux = (fluxCSV: string): FromFluxResult => {
     const result = {
       table,
       fluxGroupKeyUnion: Array.from(fluxGroupKeyUnion),
+      resultColumnNames: Array.from(resultColumnNames),
     }
 
     return result
   } catch (error) {
     return {
+      error: error as Error,
```

Comment:
We have to use `as Error` here because the `catch` clause must take **_any_** as the type.